package.json

```diff
@@ -41,7 +41,6 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
-    "@types/axe-core": "^2.0.7",
```

Comment:
This is deprecated as axe-core provides it's own type definitions so I went ahead and removed it as well.